app/layouts/DashboardLayout/OnboardingModal.view/OnboardingModal.test.tsx

```diff
@@ -7,7 +7,7 @@ import '../../../testUtils/i18nForTests';
 import { OnboardingModal } from './OnboardingModal.view';
 
 const confirmEntropyKnown = jest.fn();
-const setPin = jest.fn();
+const updatePin = jest.fn();
```

Comment:
this and all other test changes are done so this directory will let me add a commit. All of our tests need a redo